_deployment/configuration/universal_tools.md

```diff
@@ -20,32 +20,50 @@ previous year. The three tables of concern are:
     * For example, IF <`Language`> IS <`ENU-Braille`> THEN <`Masking`> IS <`TDS_Masking0` (disabled)>
         * In other words, if "Braille" is selected as the language, then the masking tool should be disabled by default
 
-The simplest way to configure accommodations/tools for a new assessment that does not have any tools configured is to select an existing
-assessment that has the test tools defined that you wish to include, and copy them over to the new assessment. This can be
-achieved by taking the following steps:
-
-1. Identify the assessment id (known as the `testid` in the configuration database)
-```
-SELECT testid FROM configs.client_testproperties WHERE label = 'IRP CAT Grade 7 MATH';
->> SBAC-IRP-Mathematics-7
-```
-2. Copy the test tool data for the testid we identified
-```
-INSERT INTO configs.client_testtooltype
-SELECT * FROM configs.client_testtooltype
-WHERE context = 'SBAC-IRP-Mathematics-7';
-```
-```
-INSERT INTO configs.client_testtool
-SELECT * FROM configs.client_testtool
-WHERE context = 'SBAC-IRP-Mathematics-7';
-```
-```
-INSERT INTO configs.client_tooldependencies
-SELECT * FROM configs.client_tooldependencies
-WHERE context = 'SBAC-IRP-Mathematics-7';
-```
-3. Flush the redis cache
+
+## Test Tool Configuration Stored Procedures
```

Comment:
Can we link to the document that has all the possible accommodation codes?

Reply:
Oh, so this enables all accommodations rather than specific ones?

Reply:
That's correct Todd. The "General Tools" inserts all "universal tools" and a few accommodations and designated supports. The `configs.client_testtooltype` table does cascaded deletes to the `configs.client_testtool` and `configs.client_tooldependencies` tables, so if we wanted to remove a specific tool type, we'd do a `DELETE FROM configs.client_testtooltype WHERE context = '<assessment id>' AND clientname = '<client>' AND toolname = '<tool name>';`. That would remove all tool options and tool dependencies.

Reply:
The ISAAP codes link is in the "## Configuring Universal Tools/Designated Supports/Accommodations for Specific Assessments and Segments" already. Do you want me to include it in this other segment as well?

Reply:
No, I didn't realize that was already there.  I pulled this down and looked at it in MacDown which is much easier.  My bad.

I didn't see a part in the documentation where it is explained how to remove the tools. Basically what you have in the comment above.  Can you add a section for that as well?  I think the common operating procedure will be to run the new stored procedures and then remove the ones that aren't needed.  Unless I missed that part in the documentation?